code/ecdsap256sha2/Hacl.Impl.LowLevel.fst

```diff
@@ -21,7 +21,7 @@ open FStar.Tactics.Canon
 open Spec.P256.Lemmas
 open Lib.IntTypes.Intrinsics
 
-#set-options "--fuel 0 --ifuel 0 --z3rlimit 500"
+#set-options "--fuel 0 --ifuel 0 --z3rlimit 1000"
```

Comment:
```suggestion
#set-options "--fuel 0 --ifuel 0 --z3rlimit 500"
```
This was fixed by another PR. Can you revert the change?

Reply:
Joy. Of course if I revert this, the thing no longer verifies on my laptop.
And if I remove hints, Poly1305.Lemmas no longer verifies on the CI machine.
When we are working with fragile proofs, let's not be too hasty to insist on dropping z3rlimits or removing hints. The workflow for getting a working PR through is already quite painful and time-consuming.

Reply:
Ok, reverted this, and got CI green, yay.